src/xcode/ENA/ENA/Source/Services/DownloadedPackagesStore/V2/DownloadedPackagesStoreV2.swift

```diff
@@ -56,6 +56,15 @@ protocol DownloadedPackagesStoreV2: AnyObject {
 	/// - Throws: An error of type `SQLiteStoreError`
 	func delete(packages: [SAPDownloadedPackage]) throws
 
+	// MARK: - Package Invalidation
+
+	/// A list of invalid ETags to be removed/ignored. Defined in the AppConfiguration.
+	var revokationList: [String] { get set }
```

Comment:
Why is is the revokationList part of the PackagesStore?
Shouldn't it be enough to just clean them from cache every time we get them?

Reply:
Checking just on AppConfig update leaves a gap in which packages might arrive that are on the 'bad list' . Keeping this list at hand for future save (`set(:)`) operations will cover cases where a download happens after the cache validation. 

I have to admit, it's an edge case but by the price of this property we cover all states regardless the order of fetch and update app config.